system/services/assetManager/AssetManagerService.cfc

```diff
@@ -936,6 +936,29 @@ component {
 		return settings;
 	}
 
+	public any function listEditorDerivates(){
+		var derivatives = _getConfiguredDerivatives();
+		var publicDerivatives = structNew();
+		var value = [];
+		var dimension = [];
+
+		for( var derivative in derivatives ) {
+
+			if ( derivatives[ derivative ].keyExists( "inEditor" ) ) {
+
+				var inEditor = derivatives[derivative].inEditor ?: false;
+
+				if( inEditor ){
+
+				    publicDerivatives.value     = value.append( derivative );
+				    publicDerivatives.dimension = dimension.append( _getPreconfiguredDerivativeTransformations( derivative ) );
```

Comment:
Given the previous note around not wanting dimensions - we should just build an array of "publicDerivatives" so this can just be:

    publicDerivatives.append( derivative );